src/app.module.ts

```diff
@@ -35,13 +35,15 @@ import { LoggerModule } from "./loggers/logger.module";
 
 @Module({
   imports: [
-    AttachmentsModule,
-    AuthModule,
-    CaslModule,
-    CommonModule,
     ConfigModule.forRoot({
       load: [configuration],
+      isGlobal: true,
+      cache: true,
```

Comment:
Does this section of code makes the configuration available everywhere without importing it in the individual modules?

Reply:
Let's make sure that we document this both in code and in documentation. Regina should be aware of this.